docs/data-sources/ram_shared_resources.md

```diff
@@ -0,0 +1,81 @@
+---
+subcategory: "Resource Access Manager (RAM)"
+---
+
+# huaweicloud_ram_shared_resources
+
+Use this data source to get the list of RAM shared resources.
+
+## Example Usage
+
+```hcl
+variable "resource_type" {}
+variable "principal" {}
+variable "resource_urns" {}
+variable "resource_share_ids" {}
+variable "resource_ids" {}
+variable "resource_region" {}
```

Comment:
it's unnecessary to list all parameters in the example, just add a basic usage will be fine.